### DISCUSSION TITLE
Record key press in image-button-response

### DISCUSSION THREAD

**Discussion by wangzhuoranivy** [jspsych/jsPsych#927] [Q&A] [answered]
Hi,

Is it still possible to detect a certain key press if the trial type is a button response?

Thanks in advance! 

Ivy

**Comment by jodeleeuw** [comment, marked as answer]
Hi Ivy,

You'd need to add this to the plugin. I recorded [a tutorial video](https://www.youtube.com/watch?v=XQcsFwAmbiw) where I develop a new plugin that uses a keyboard response. You could watch it to learn how to add keyboard responses to plugins, and then add it to the button response plugin.

**Reply by wangzhuoranivy** [reply]
I will go check it out. Thanks!
